frontend/src/components/about.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
Based off of your imports, it looks like you renamed all of your components to their capitalized name. about.jsx -> About.jsx. 

One annoying thing about React & github is that it actually doesn't automatically recognize a capitalization change as a file change, so github doesn't have the updated capitalized names of files, which means that someone who is trying to run your code will be greeted with a few errors along the line of "the component at this route does not exist." 

Check out this github for more info. [https://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git](url)

I went ahead and locally fixed these capitalization errors so I could test/review your code